mne/epochs.py

```diff
@@ -1913,7 +1920,8 @@ def __init__(self, data, info, events, tmin=0, event_id=None,
         super(EpochsArray, self).__init__(info, data, events, event_id, tmin,
                                           tmax, baseline, reject=reject,
                                           flat=flat, reject_tmin=reject_tmin,
-                                          reject_tmax=reject_tmax, decim=1)
+                                          reject_tmax=reject_tmax, decim=1,
+                                          add_eeg_ref=False)
```

Comment:
We were silently always adding an `EEG` ref before but not applying it (whoops). So we could consider this a bug fix...?